### PR TITLE
Feature US10

### PR DESCRIPTION
**Pull request by robinschubertdev**

### CODE REVIEW

File: components/EventForm.js [hunk 1]
```diff
@@ -17,9 +19,40 @@ const StyledFieldset = styled.fieldset`
...
+  const currentStep = formStep;
+
+  const [startDateTime, setStartDateTime] = useState();
+  const [endDateTime, setEndDateTime] = useState();
+
+  const nextFormStep = () => setFormStep(formStep + 1);
+  const prevFormStep = () => setFormStep(formStep - 1);
+
+  const todayISO = new Date().toISOString();
+  const today = todayISO.slice(0, -8);
```
Comment (moonwave99) [inline components/EventForm.js:52]: Can be just `const today = new Date().toISOString().slice(0, -8)`

File: components/EventForm.js [hunk 2]
```diff
@@ -17,9 +19,40 @@ const StyledFieldset = styled.fieldset`
...
+
+  console.log(todayISO, today);
```
Comment (moonwave99) [inline components/EventForm.js:54]: no logs

File: components/EventForm.js [hunk 3]
```diff
@@ -34,10 +67,13 @@ export default function EventForm() {
 
   return (
     <>
+      <ProgressBar steps={"4"} currentStep={currentStep} />
```
Comment (moonwave99) [inline components/EventForm.js:70]: `<ProgressBar steps={4}>`

File: components/EventForm.js [hunk 4]
```diff
@@ -107,58 +135,94 @@ export default function EventForm() {
...
+        </>
+
+        <ButtonContainer>
+          {formStep > 0 && <Button onClick={prevFormStep}>Back</Button>}
+          {formStep < 3 && <Button onClick={nextFormStep}>Next</Button>}
+          {formStep === 3 && (
+            <Button onClick={nextFormStep}>Check all entered Data</Button>
+          )}
+          {formStep === 4 && (
+            <Button color={"green"} type="Submit">
```
Comment (moonwave99) [inline components/EventForm.js:221]: `<Button color="green" type="submit">`

File: pages/events/create/index.js [hunk 5]
```diff
@@ -4,7 +4,6 @@ import LinkButton from "@/components/LinkButton";
 export default function Page() {
   return (
     <>
```
Comment (moonwave99) [inline pages/events/create/index.js:6]: No diamonds with a single child ^^

**Review by moonwave99 — COMMENTED** [review]
Nicely done!

I would say at the beginning of the file:

```js
const STEP_COUNT = 4;
```

And replace all the occurrences of the number 4 in the file, so it looks less "magic"

**Comment by fel-sad** [comment]
Thank your for the lightning fast review! 🚀  We just implemented your suggestions.

**Pull request merged**